src/Tabs/README.md

```diff
@@ -0,0 +1 @@
+# Tabs
```

Comment:
Probably don't need a nested README if we're following Steve's convention. He places all of these in the root README.